publication-commons/src/main/java/no/unit/nva/publication/model/storage/DoiRequestDao.java

```diff
@@ -110,12 +109,6 @@ public String toString() {
         return toJsonString();
     }
 
-    private TransactWriteItem createUniqueDoiRequestEntry() {
-        UniqueDoiRequestEntry uniqueDoiRequestEntry = new UniqueDoiRequestEntry(
-            getTicketEntry().getResourceIdentifier().toString());
-        return newPutTransactionItem(uniqueDoiRequestEntry);
-    }
-
```

Comment:
Removes constraints for one DoiRequest per publication. 
If user decides to remove DoiRequest for some reason -> Just request new one.